ui/app/src/modules/Preview/PreviewConcierge.tsx

```diff
@@ -180,6 +182,9 @@ export function PreviewConcierge(props: any) {
       case 'craftercms.ice.assets':
         // TODO: aaron to fetch assets here...
         break;
+      case 'craftercms.ice.audiences':
+        (audiencesPanel === null && site) && dispatch(fetchAudiencesPanelConfig());
```

Comment:
You probably will end up with a non-null root state branch so do revisit this condition.